Makefile

```diff
@@ -1,6 +1,8 @@
 CORE = rel/epoch/bin/epoch
 VER := $(shell cat VERSION)
 
+REBAR ?= ./rebar3
```

Comment:
Would it perhaps be even better with
`REBAR ?= $(shell which rebar3 || echo ./rebar3)`?

(Though I don't know how that works on Windows...)